### PR TITLE
More Sphinx doc

### PR DESCRIPTION
**Pull request by Fabien-B**
More user oriented sphinx doc.
Checkout the rendered doc here : https://fabien-b.github.io/paparazzi/

We can also discuss the plan, I'm not so sure about the current one.
Maybe I would add a "Tools" section, with a page for each tool, and "Modules" section for airborne modules.

### CODE REVIEW

**Comment by gautierhattenberger** [comment]
it seems to have conflicts with master

**Comment by Fabien-B** [comment]
I am not sure what happened with theses conflicts. Nobody else modified the sphinx doc so I squashed all the diff with master in one commit.

File: doc/sphinx/source/user_guide/directories_structure.rst [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    │   │   └── messages.xml
+    ├── doc
+    │   └── sphinx
+    │       └── source
+    ├── Makefile
+    ├── paparazzi
+    ├── start.py
+    ├── sw
+    │   ├── ext
+    │   └── airborne
```
Comment (gautierhattenberger) [inline doc/sphinx/source/user_guide/directories_structure.rst:47]: add `ground_segment/tmtc` `ground_segment/cockpit` and `simulator` folders at least ?

File: doc/sphinx/source/user_guide/directories_structure.rst [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+-------
+
+Modules configuration files are located in the ``conf/modules`` directory. The actual code of the modules is located in ``sw/airborne/modules``.
+
+
+Software
+--------
+
+Paparazzi softwares are located in ``sw``. Airborne code, that will run on the drone itself, is located in ``sw/airborne``.
+
```
Comment (gautierhattenberger) [inline doc/sphinx/source/user_guide/directories_structure.rst:110]: add `tmtc`, `cockpit` and `simulator` location and content

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]

File: doc/sphinx/source/user_guide/directories_structure.rst (hunk 2 above)
Comment (Fabien-B) [reply doc/sphinx/source/user_guide/directories_structure.rst:114]: Can you suggest modification for the `tmtc` directory? I never understood the common ground of this directory.

File: doc/sphinx/source/quickstart/install.rst [hunk 3]
```diff
@@ -8,46 +8,69 @@ Paparazzi runs best on **Ubuntu 16.04 or higher**, so this quick installation gu
...
+**If you have Ubuntu 20.04:**
+
+.. code-block:: bash
+
+    sudo apt-get install python-is-python3
+    wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2
+    sudo tar -xvf gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2 -C /usr/local
+    sudo ln -s /usr/local/gcc-arm-none-eabi-7-2017-q4-major /usr/local/gcc-arm-none-eabi
+    echo "export PATH=\"\$PATH:/usr/local/gcc-arm-none-eabi/bin\"" >> .bashrc
+    source .bashrc
```
[the remark this answers is not in the document]
Comment (Fabien-B) [reply doc/sphinx/source/quickstart/install.rst:23]: Is `ppa:team-gcc-arm-embedded/ppa` available now for 20.04 ?
One thing is sure (at least is sure up to 18.04) : the `gcc-arm-none-eabi` packet in the ubuntu ppa is flawed. Software build but the binary produced does not work.

File: doc/sphinx/source/user_guide/directories_structure.rst (hunk 2 above)
Comment (gautierhattenberger) [reply doc/sphinx/source/user_guide/directories_structure.rst:114]: it stands for "telemetry and telecommand", so mostly tools related to modems (link) and the server

File: doc/sphinx/source/quickstart/install.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (gautierhattenberger) [reply doc/sphinx/source/quickstart/install.rst:23]: The standard gcc-arm-none in Ubuntu 20.04 is now the one packaged by ARM, so it is okay to use it. For previous Ubuntu version, the launchpad version is recommended.

**Comment by Fabien-B** [comment]
Changes made, I added in the install page the packet `gdb-multiarch` for ubuntu 20.04, needed to debug or flash a board via SWD. (not packaged with gcc-arm-embedded)

File: doc/sphinx/source/quickstart/install.rst [hunk 4]
```diff
@@ -8,46 +8,63 @@ Paparazzi runs best on **Ubuntu 16.04 or higher**, so this quick installation gu
...
 
 .. code-block:: bash
 
-	sudo add-apt-repository -y ppa:paparazzi-uav/ppa
-	sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa
-	sudo apt-get update
-	sudo apt-get -f -y install paparazzi-dev paparazzi-jsbsim gcc-arm-embedded
+    sudo add-apt-repository -y ppa:paparazzi-uav/ppa
+    sudo apt-get update
+    sudo apt-get -f -y install paparazzi-dev paparazzi-jsbsim gcc-arm-embedded
```
Comment (gautierhattenberger) [inline doc/sphinx/source/quickstart/install.rst:36]: `gcc-arm-embedded` is the name of the package from ppa, but when installed from ubuntu it is `gcc-arm-none-eabi`, so I think it should be in the specific part.

**Pull request merged**